source/reporters/ut_realtime_reporter.tpb

```diff
@@ -57,25 +57,27 @@ create or replace type body ut_realtime_reporter is
       self.print_node('name', a_suite.name);
       self.print_node('description', a_suite.description);
       <<suite_elements>>
+      self.print_start_node('items');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 4 times. ([plsql:S1192](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=819&issues=AWhDYNbDXONnvcWDUk4g&open=AWhDYNbDXONnvcWDUk4g)

[ISSUE_KEY]: AWhDYNbDXONnvcWDUk4g